Makefile

```diff
@@ -96,6 +97,14 @@ create-app-symlink:
 	cd lab; \
 	sudo clab exec -t $(LABFILE) --label clab-node-kind=srl --cmd 'sudo ln -s /opt/$(APPNAME)/run.sh /usr/local/bin/$(APPNAME)'
 
+update-appmgr-dir:
+	cd lab; \
+	sudo clab exec -t $(LABFILE) --label clab-node-kind=srl --cmd 'sudo bash -c "mkdir -p /etc/opt/srlinux/appmgr && cp /tmp/$(APPNAME).yml /etc/opt/srlinux/appmgr/$(APPNAME).yml"'
```

Comment:
any particular reason you decided to carve out file moving to a separate make target?
It was done inside `create-app-symlink` in Go https://github.com/srl-labs/ndk-dev-environment/pull/7/files#diff-76ed074a9305c04054cdebb9e9aad2d818052b07091de1f20cad0bbac34ffb52R81

Reply:
> any particular reason you decided to carve out file moving to a separate make target? It was done inside `create-app-symlink` in Go https://github.com/srl-labs/ndk-dev-environment/pull/7/files#diff-76ed074a9305c04054cdebb9e9aad2d818052b07091de1f20cad0bbac34ffb52R81

I removed `create-app-symlink` to make possible the python app deployment with a single RPM install command.